### DISCUSSION TITLE
--no-warn-ignored does not work?

### DISCUSSION THREAD

**Discussion by zpinto123** [eslint/eslint#18019] [Q&A] [answered]
Hey guys,

I currently have eslint working normal, but if I try and use the new --no-warn-ignored argument, I'll get this error:
**Invalid option '--warn-ignored' - perhaps you meant '--ignore'?**

Tried all these 3 different commands, but they all give the same error.
```
npx eslint --no-warn-ignored
npx eslint --max-warnings 0 --no-warn-ignored
npx eslint@8.51.0 --max-warnings 0 --no-warn-ignored
```

My eslint is version 8.56.0.
Even when running the 8.51.0 version, I get this error (version where --no-warn-ignored was introduced).

**Comment by mdjermanovic** [comment, marked as answer]
Hi @zpinto123! 

Are you using `eslint.config.js` (the new flat config format), or `.eslintrc` config files? This CLI option is available only when using [`eslint.config.js`](https://eslint.org/docs/latest/use/configure/configuration-files-new) config files.

**Reply by zpinto123** [reply]
Hmm, that must be the issue then.
I'm now trying to convert to `eslint.config.js`, but I noticed that it no longer supports a few of the properties that I have, like "env", "extends", etc.
Is there somewhere I can check on what to do about these properties that are no longer supported?

**Reply by nzakas** [reply]
https://eslint.org/docs/latest/use/configure/migration-guide

**Reply by zpinto123** [reply]
Thank you
